Remove version check which can cause problems for long running
processes embedding perl when upgrading to a newer version,
compatible, but built on a different machine.

diff -Naur --exclude=debian perl-5.10.0.orig/ext/Errno/Errno_pm.PL perl-5.10.0/ext/Errno/Errno_pm.PL
--- perl-5.10.0.orig/ext/Errno/Errno_pm.PL	2007-11-18 02:26:24.000000000 +1100
+++ perl-5.10.0/ext/Errno/Errno_pm.PL	2007-11-26 07:17:26.000000000 +1100
@@ -337,13 +337,8 @@
 package Errno;
 our (\@EXPORT_OK,\%EXPORT_TAGS,\@ISA,\$VERSION,\%errno,\$AUTOLOAD);
 use Exporter ();
-use Config;
 use strict;
 
-"\$Config{'archname'}-\$Config{'osvers'}" eq
-"$Config{'archname'}-$Config{'osvers'}" or
-	die "Errno architecture ($Config{'archname'}-$Config{'osvers'}) does not match executable architecture (\$Config{'archname'}-\$Config{'osvers'})";
-
 \$VERSION = "$VERSION";
 \$VERSION = eval \$VERSION;
 \@ISA = qw(Exporter);
